apps/ares-client/helpers/auth/login.ts

```diff
@@ -1,32 +1,28 @@
 import { UserContext } from "@/types/user/user";
-import { createContext, useContext } from "react";
+import { createContext } from "react";
 
 export const User = createContext<UserContext>({
   user: undefined,
   updateUser: () => {},
 });
 
 export const login = async (name: string, password: string) => {
```

Comment:
unnecessary try catch

Reply:
testing errors :D

Reply:
this is becoming a real thing lol @nazmik @vascularone

Reply:
mos mu bo more alek ti me rq, bone approve pot tqi nanen

Reply:
kur tmarrim puntor tri ja bon atynve what the fuck is this :D @nazmik @vascularone

Reply:
boni add back try and catch

Reply:
Mos ha car npushim jom 🤒